v_3_6_2/master_template.go

```diff
@@ -153,6 +153,7 @@ write_files:
             - effect: NoExecute
               operator: Exists
           serviceAccountName: calico-node
+          priorityClassName: system-cluster-critical
```

Comment:
Please add this to calico manifest changes, otherwise it will be probably gone on nxt update.

Reply:
Where is calico manifest changes?

Reply:
Should we also add this in azure-operator manifests?

Reply:
Yes definitely. We should make sure to use the same source for both imo. Do we have an issue for that already?

Reply:
probably yes, I am kinda shady on where is that

Reply:
using same source fro both can be tricky as azure uses completely different calico ( AFAIK)

Reply:
then we maybe should put a big fat warning in here. and add a link to the one in the azure operator.

Reply:
Yeah make sense

Reply:
Changes are tracked in comments here https://github.com/giantswarm/k8scloudconfig/blob/master/v_3_7_1/master_template.go#L25

We have calico in many places, so we should be careful, i found most reliable way is to track deviations from upstream manifests in comments. Better ideas are welcome

Reply:
https://github.com/giantswarm/k8scloudconfig/pull/457